python/main.py

```
@@ -199,6 +202,8 @@ async def main():
     else:
         print("No even numbers found")
     
+    print("This will cause a syntax error: " + (5 + 3 * 2
```

Pylint (3.3.7)

[error] Parsing failed: ''(' was never closed (main, line 205)'

(E0001)